R/count_teleconnections.R

```diff
@@ -201,7 +201,7 @@ count_watershed_teleconnections <- function(data_dir,
         if (city %in% sliced_cities) {
           get_raster_val_classes_byslice(watersheds_city, economic_USA, poly_slices, n_cores) -> economic_ids
```

Comment:
Same comment here about not needing the sliced cities conditional statement

Reply:
@crvernon We were still experiencing some slow downs in the process and found it was due to coercing the list (created from exact_extractr) to a dataframe. We are now having memory issues and are trying to lower the resolution of the large watersheds in order for them to run more efficiently. Working on this now.